assets/data/cards_pt-br.json

```diff
@@ -175,6 +175,19 @@
       "title": "Node.js",
       "description": "Node.js é um ambiente que permite executar JavaScript no servidor. Isso significa que podemos usar JavaScript para criar e rodar aplicações do lado do servidor.",
       "tags": ["Back-end"]
+    },
+    {
```

Comment:
Isso aqui já estava quando eu fiz meu PR, só estou "colocando de volta" porque foi removido por algum motivo nesse meio tempo.

Reply:
Nvm, to removendo, porque os dois já estão presentes em outro ponto do arquivo.
[Seria bom pedir pra sempre adicionar no final.]